newsService/src/main/java/com/kovan/api/model/TestRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.kovan.api.model;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+
+@Data
+@Document(collection = "TestData")
```

Comment:
why do we need this ?

Reply:
Removed @Document(collection = "TestData") from TestRequest class.